### PR TITLE
Remove the brand icons

### LINKED ISSUE #15284: Remove brand icons from Help Menu
The icons were added in 

* https://github.com/ankidroid/Anki-Android/pull/8519

These were using a "GPL Friendly" license from Pictogrammers

It turns out that the license wasn't GPL-Friendly, specifically regarding brand icons: https://github.com/Templarian/MaterialDesign/issues/5777

and brand icons were removed in V6: https://github.com/Templarian/MaterialDesign/issues/6602

The logical conclusion to me would be to remove the brand icons if they're nonfree

_Originally posted by @david-allison in https://github.com/ankidroid/Anki-Android/issues/15255#issuecomment-1900742846_

### PR DESCRIPTION
**Pull request by criticalAY**
## Purpose / Description
Removing the brand icons due to not being free and replacing them with simple link icon

* Fixes #15284

## How Has This Been Tested?
Google emulator
image
 

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/res/drawable/twitter.xml [hunk 1]
```diff
@@ -1,32 +1,12 @@
...
-# Fonts: Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0)
-All web and desktop fonts are distributed under the Apache 2.0 license. Web
-and desktop fonts contain some icons that are redistributed under the Apache
-2.0 license. All other icons are either redistributed under their respective
-licenses or are distributed under the Apache 2.0 license.
-
-# Code: MIT (https://opensource.org/licenses/MIT)
-The MIT license applies to all non-font and non-icon files.
- -->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (david-allison) [inline AnkiDroid/src/main/res/drawable/twitter.xml:1]: What's the license?

Comment (criticalAY) [reply AnkiDroid/src/main/res/drawable/twitter.xml:1]: The OP put it here i.e the source they downloaded it from would have it there, this icon that I updated doesn't need it, it's free and attribution is optional so I skipped it and I made some changes to the logo color and dimensions so it technically belongs to me now 😁, and I don't wish to put attribution here

Comment (david-allison) [reply AnkiDroid/src/main/res/drawable/twitter.xml:1]: Sorry, I don't understand?

Where did the file come from and what license was it under before you made modifications?

Comment (criticalAY) [reply AnkiDroid/src/main/res/drawable/twitter.xml:1]: The previous file was downloaded from http://pictogrammers.com where they would have put these licenses, wait I will update it

**Comment by criticalAY** [comment]
I have made some changes, now it has nothing to do with the previous icon file

File: AnkiDroid/src/main/res/drawable/twitter.xml (hunk 1 above)
Comment (david-allison) [reply AnkiDroid/src/main/res/drawable/twitter.xml]: It's fine (& encouraged) to keep the file in the same location

But the commit says nothing about the source of the current icon, and the copyright header doesn't appear accurate

* The commit message should ideally list where the icon came from
* If you made changes (especially if the file is Apache)
  * You need to list the changes which are made
  * It's likely OK to post a link to the original file, and have your commit contain the changed icon
  * It's less ambiguous to commit the original file, and then commit your changes as a separate commit on top of this

**Comment by david-allison** [comment]
The icons were added in 

https://github.com/ankidroid/Anki-Android/pull/8519

These were using a "GPL Friendly" license from Pictogrammers

It turns out that the license wasn't GPL-Friendly, specifically regarding brand icons: https://github.com/Templarian/MaterialDesign/issues/5777

and brand icons were removed in V6: https://github.com/Templarian/MaterialDesign/issues/6602

The logical conclusion to me would be to remove the brand icons if they're nonfree

----

Pending a further discussion on Discord

**Comment by lukstbit** [comment]
> The logical conclusion to me would be to remove the brand icons if they're nonfree

I think this is the way to go, icons would be nice but definitely not worth the hassle.

**Comment by mikehardy** [comment]
Agreed

Remove the brand icons

**Comment by david-allison** [comment]
@lukstbit would you like to include removal in https://github.com/ankidroid/Anki-Android/pull/15190, or have someone split it out into a separate PR?

**Review by david-allison — APPROVED** [review]

**Review by lukstbit — CHANGES_REQUESTED** [review]
Looks good, however I think it could be better by:
- swapping the _Reddit_ and _Mailing List_ so all social networks links are together 
- using the link icon for _Reddit_ as well like it was used for the other social networks

**Review by lukstbit — APPROVED** [review]
Fantastic, thanks!

**Pull request merged**